include/grpcpp/impl/codegen/call_op_set.h

```diff
@@ -485,10 +484,19 @@ class CallOpRecvMessage {
   }
 
  private:
-  R* message_;
+  // Sets got_message and \a status for a failed recv message op
+  void FinishOpRecvMessageFailureHandler(bool* status) {
+    got_message = false;
+    if (!allow_not_getting_message_) {
+      *status = false;
+    }
+  }
+
+  R* message_ = nullptr;
   ByteBuffer recv_buf_;
-  bool allow_not_getting_message_;
+  bool allow_not_getting_message_ = false;
   bool hijacked_ = false;
+  bool hijacked_recv_message_status_ = true;
```

Comment:
This is odd to be true by default without a comment explaining what's going on here.

Reply:
changed it to hijacked_recv_message_failed_ so that it is only true when the op failed when hijacking